bust-a-move-framework/src/main/java/bustamove/bubble/BubbleStorage.java

```diff
@@ -191,14 +193,9 @@ public boolean isFull() {
      */
     public void removeBubble(final Bubble bubble) {
         for (Bubble[] row : bubble2DArray) {

```

Comment:
It will now continue to search after it has removed the bubble. The finished variable is to prevent this